includes/modules/debug/assets/js/debug.js

```diff
@@ -0,0 +1,128 @@
+var Debug = function() {
+	var self = this,
+		errorStack = [],
+		settings = {},
+		elements = {};
+
+	var initSettings = function() {
+		settings = {
+			debounceDelay: 500,
+			urlsToWatch: [
```

Comment:
Please add a method to add a Watch Url:

    addWatchUrl: function( url ) {
        var urls = this.getSettings( 'urlsToWatch' );
        urls.push( url );
    },